internal/db/remote/changes/changes.go

```diff
@@ -170,25 +170,9 @@ EOSQL
 			return errors.New("Error starting shadow database: " + errBuf.String())
 		}
 
-		{
-			out, err := utils.DockerExec(ctx, dbId, []string{
-				"sh", "-c", `PGOPTIONS='--client-min-messages=error' psql postgresql://postgres:postgres@localhost/postgres <<'EOSQL'
-BEGIN;
-` + utils.InitialSchemaSql + `
-COMMIT;
-EOSQL
-`,
-			})
-			if err != nil {
-				return err
-			}
-			var errBuf bytes.Buffer
-			if _, err := stdcopy.StdCopy(io.Discard, &errBuf, out); err != nil {
-				return err
-			}
-			if errBuf.Len() > 0 {
-				return errors.New("Error starting shadow database: " + errBuf.String())
-			}
+		p.Send(utils.StatusMsg("Resetting database..."))
+		if err := differ.ResetDatabase(ctx, dbId, utils.ShadowDbName); err != nil {
```

Comment:
Should be `postgres` which is the default `POSTGRES_DB`
```suggestion
		if err := differ.ResetDatabase(ctx, dbId, "postgres"); err != nil {
```
We might also need `PGOPTIONS='--client-min-messages=error` in `reset.sh`

Reply:
> Should be postgres which is the default POSTGRES_DB

This is a new change introduced in `14.1.0.66`: [docker entrypoint](https://github.com/supabase/postgres/blob/develop/ansible/playbook-docker.yml#L21) now creates additional roles and schemas in default `postgres` db. Our initial schema tries to create the same schema and fails. Hence, we use shadow db. Also, this change makes remote diff consistent with local diff.